ui/raidboss/data/07-dt/dungeon/mistwake.txt

```diff
@@ -0,0 +1,226 @@
+2134.6 "--sync--" #Ability { id: "B124", source: "Amdusias" }
+2136.2 "--sync--" #Ability { id: "B124", source: "Amdusias" }
+2137.6 "--sync--" #Ability { id: "B124", source: "Amdusias" }
+2140.0 "Galloping Thunder" Ability { id: "B122", source: "Amdusias" }
+2140.2 "Galloping Thunder" #Ability { id: "B123", source: "Amdusias" }
+2142.3 "Galloping Thunder" #Ability { id: "B123", source: "Amdusias" }
+2144.4 "Galloping Thunder" #Ability { id: "B123", source: "Amdusias" }
+2146.5 "Galloping Thunder" #Ability { id: "B123", source: "Amdusias" }
+2148.6 "Galloping Thunder" #Ability { id: "B123", source: "Amdusias" }
+2154.2 "Thunder IV" Ability { id: "B126", source: "Amdusias" } forcejump "amdusias-loop"
+
+#~~~~~~~~~~~~~~~~~~~~~#
+# Thundergust Griffin #
+#~~~~~~~~~~~~~~~~~~~~~#
+
+# -ii B09C B09F B0A1 B0E8 B0ED B0F0 B0F1 B0F2 B0FB B0FC B75F B760 B761
+
+# IGNORED ABILITIES
+# B09C --sync--
+# B09F --sync--
+# B0A1 --sync--
+# B0E8 Mistic Ruin
+# B0ED Lightning Bolt - Paired with B708
+# B0F0 Thunderbolt
+# B0F1 Thunderbolt
+# B0F2 Thunderbolt
+# B0FB attack
+# B0FC attack
+# B75F --sync--
+# B760 --sync--
+# B761 --sync--
+
+# ALL ENCOUNTER ABILITIES
+# B0EB Thunderspark - Aoe, sets up deathwall
+# B0EC High Volts - Spread, sets up lightning orbs
+# B0EE --sync--
+# B0EF Thundering Roar - Activates lightning orbs
+# B0F3 Storm Surge - Spawn lightning cloud on edge
+# B0F4 Electrifying Flight
+# B0F5 Fulgurous Fall - Knockback out of center
+# B0F6 Rush - Related to Fulgurous Fall
+# B0F8 Electrogenetic Force - Line aoe on return
+# B0F9 Golden Talons - Tank buster
+# B704 --sync-- - ??
+# B708 Lightning Bolt - Places lightning orbs
+# B7AE Electrifying Flight - ???
+
+# TODO: get a longer pull and confirm loop
+
+# The Town Round will be sealed off
+3000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "14EC" } window 3000,1
+# Sync up to first cast
+3006.1 "--sync--" StartsUsing { id: "B0EB", source: "Thundergust Griffin" }
+3011.1 "Thunderspark" Ability { id: "B0EB", source: "Thundergust Griffin" }
+3016.3 "--sync--" Ability { id: "B704", source: "Thundergust Griffin" }
+3022.9 "High Volts" Ability { id: "B0EC", source: "Thundergust Griffin" }
+3023.9 "Lightning Bolt" Ability { id: "B708", source: "Thundergust Griffin" }
+3027.0 "--sync--" Ability { id: "B0EE", source: "Thundergust Griffin" }
+3034.0 "Thundering Roar" Ability { id: "B0EF", source: "Thundergust Griffin" }
+3037.8 "--sync--" Ability { id: "B0EE", source: "Thundergust Griffin" }
+3044.8 "Thundering Roar" Ability { id: "B0EF", source: "Thundergust Griffin" }
+3056.6 "Golden Talons" Ability { id: "B0F9", source: "Thundergust Griffin" }
+3063.7 "--sync--" Ability { id: "B704", source: "Thundergust Griffin" }
+3072.8 "Fulgurous Fall" Ability { id: "B0F5", source: "Thundergust Griffin" }
+3072.8 "Rush" Ability { id: "B0F6", source: "Thundergust Griffin" }
+3072.9 "Electrifying Flight" Ability { id: "B7AE", source: "Thundergust Griffin" }
```

Comment:
It just now occurred to me, could you re-run `make_timeline` with `-it "Thundergust Griffin"` and include the targetable timings here and elsewhere?

Reply:
Good point, I'll take a look.